### PR TITLE
Feature/ecrm domain modifications

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

**Comment by brettminnie** [comment]
Some stuff that's not required here, let me fix it

**Comment by eddiejaoude** [comment]
Ok

**Comment by brettminnie** [comment]
Right fixed that, good thing you moaned about the size :smile:

**Comment by eddiejaoude** [comment]
:grin:

File: src/TransformCore/PHE/HayApi/PersistenceBundle/Entity/Address.php [hunk 1]
```diff
@@ -37,19 +37,19 @@ class Address implements IsIdentifiable
...
      */
-    protected $county;
+    protected $county = '';
 
     /**
      * @ORM\Column(type = "string", nullable = true)
      * @var string
      */
-    protected $district;
+    protected $district = '';
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Entity/Address.php]: If it defaults to an empty string (`''`), does it need to be `nullable = true`?

**Comment by eddiejaoude** [comment]
> ...good thing you moaned about the size...

I just expressed concern :wink:

File: src/TransformCore/PHE/HayApi/PersistenceBundle/Entity/Address.php (hunk 1 above)
Comment (brettminnie) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Entity/Address.php]: No longer, I'll update

**Pull request merged**